### PR TITLE
docs: adds section on encapsulation

### PR DESCRIPTION
**Pull request by faiq**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-85911

## Description of changes being made

This adds a section explaining how to change the encapsulation method for preprovisioned clusters.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4455.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 1]
```diff
@@ -158,6 +160,51 @@ Change the value for `spec.calicoNetwork.nodeAddressAutodetectionV4` to `interfa
...
+### Changing encapsulation type
+
+Calico can leverage different network encapsulation methods to route traffic for your workloads. Encapsulation can be useful when running on top of an underlying network that cannot easily be made aware of workload IPs. Common examples of this include:
+
+
+    - public cloud environments where you don’t own the hardware
+    - AWS across VPC subnet boundaries
+    - environments where you cannot peer Calico over BGP to the underlay or easily configure static routes.
+
+IPIP is the default encapsualtion method.
```
Comment (cneth) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md:172]: Typo:

```suggestion
IPIP is the default encapsulation method.
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 2]
```diff
@@ -158,6 +160,51 @@ Change the value for `spec.calicoNetwork.nodeAddressAutodetectionV4` to `interfa
...
+
+The supported values are "IPIPCrossSubnet", "IPIP", "VXLAN", "VXLANCrossSubnet", and "None".
+
+#### VXLAN
+
+VXLAN is a tunneling protocol that encapsulates layer 2 Ethernet frames in UDP packets, enabling you to create virtualized layer 2 subnets that span Layer 3 networks. It has a slightly larger header than IP-in-IP which creates a slight reduction in performance over IP-in-IP.
+
+#### IPIP
+
+IP-in-IP is an IP tunneling protocol that encapsulates one IP packet in another IP packet. An outer packet header is added with the tunnel entrypoint and the tunnel exit point. The calico implementation of this protocol uses BGP to determine the exit point making this protocol unusable on networks that don’t pass BGP (eg Azure).
```
Comment (cneth) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md:197]: Seems odd to mention Azure in this page for 'pre-provisioned' infra.

```suggestion
IP-in-IP is an IP tunneling protocol that encapsulates one IP packet in another IP packet. An outer packet header is added with the tunnel entrypoint and the tunnel exit point. The calico implementation of this protocol uses BGP to determine the exit point making this protocol unusable on networks that don’t pass BGP.
```

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 3]
```diff
@@ -89,7 +89,9 @@ enterprise: false
     dkp get kubeconfig -c ${CLUSTER_NAME} > ${CLUSTER_NAME}.conf
     ```
 
-### Verify your Calico installation
+### Modifying your Calico installation
```
Comment (cmastr) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md:92]:
```suggestion
### Modify the Calico installation
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 4]
```diff
@@ -89,7 +89,9 @@ enterprise: false
...
+
+### Setting interface
```
Comment (cmastr) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md:94]:
```suggestion
### Set the interface
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 5]
```diff
@@ -158,6 +160,51 @@ Change the value for `spec.calicoNetwork.nodeAddressAutodetectionV4` to `interfa
 
 Save this file. You may need to delete the node feature discovery worker pod in the `node-feature-discovery` namespace if that pod has failed. After you delete it, Kubernetes replaces the pod as part of its normal reconciliation.
 
+### Changing encapsulation type
```
Comment (cmastr) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md:163]:
```suggestion
### Change the encapsulation type
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 6]
```diff
@@ -158,6 +160,51 @@ Change the value for `spec.calicoNetwork.nodeAddressAutodetectionV4` to `interfa
...
+
+Calico can leverage different network encapsulation methods to route traffic for your workloads. Encapsulation can be useful when running on top of an underlying network that cannot easily be made aware of workload IPs. Common examples of this include:
```
Comment (cmastr) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md:165]:
```suggestion
Calico can leverage different network encapsulation methods to route traffic for your workloads. Encapsulation is useful when running on top of an underlying network that cannot easily be made aware of workload IPs. Common examples of this include:
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 7]
```diff
@@ -158,6 +160,51 @@ Change the value for `spec.calicoNetwork.nodeAddressAutodetectionV4` to `interfa
...
+
+
+    - public cloud environments where you don’t own the hardware
+    - AWS across VPC subnet boundaries
+    - environments where you cannot peer Calico over BGP to the underlay or easily configure static routes.
+
+IPIP is the default encapsulation method.
+
+To change the encapsulation run the command:
```
Comment (cmastr) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md:174]:
```suggestion
To change the encapsulation, run the following command:
```

**Comment by cmastr** [comment]
@faiq when we get this finalized, we should put this in 2.3 as well.

**Review by cmastr — APPROVED** [review]

**Pull request merged**